storage/src/nodestore.rs

```diff
@@ -1616,9 +1713,17 @@ impl<S: ReadableStorage> NodeStore<Committed, S> {
 pub(crate) mod nodestore_test_utils {
     use super::*;
 
+    pub(crate) const fn area_sizes() -> &'static [u64] {
```

Comment:
why did you go this route instead of just making AREA_SIZES `pub(crate)`? It's already an immutable static.